libs/datamodel/core/src/transform/ast_to_dml/validate.rs

```diff
@@ -672,6 +672,13 @@ impl<'a> Validator<'a> {
                         .unwrap_or_default()
                 {
                     referential_actions::detect_cycles(datamodel, model, field, field_span, &mut errors);
+                    referential_actions::detect_multiple_cascading_paths(
+                        datamodel,
+                        model,
+                        field,
+                        field_span,
+                        &mut errors,
+                    );
```

Comment:
It's probably just not visible no the diff, but this is gated to only mssql, right?

Reply:
Take a look into a few lines above:
```rust
connector.has_capability(ConnectorCapability::ReferenceCycleDetection)
```
So if the connector "has this capability", it's run.